bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/metrics/MetricRegistry.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This `MetricRegistry.java` file was added as a new file with 85333883d3c12edcc4521ded0aab3c7c533410e5. Also IDE detects this as a new file.

![Screenshot 2021-06-14 at 19 01 44](https://user-images.githubusercontent.com/26219651/121900815-771fae80-cd43-11eb-9f5d-f38004dffdcf.png)